tools/gen_ort_dockerfile.py

```diff
@@ -441,7 +447,9 @@ def dockerfile_for_windows(output_file):
 ARG ONNXRUNTIME_VERSION
 ARG ONNXRUNTIME_REPO
 RUN git clone -b rel-%ONNXRUNTIME_VERSION% --recursive %ONNXRUNTIME_REPO% onnxruntime && \
-    (cd onnxruntime && git submodule update --init --recursive)
+    cd onnxruntime && git submodule update --init --recursive && \
+    git config --global user.email "you@example.com" && git config --global user.name "Your Name" && \
+    git cherry-pick 709368ea1443dc1ff68dece31d692ad065fb94d4
```

Comment:
Suppose we can skip this part:
```suggestion
    cd onnxruntime && git submodule update --init --recursive
```